docs-2.0/3.ngql-guide/11.edge-statements/2.update-edge.md

```diff
@@ -0,0 +1,24 @@
+# UPDATE EDGE
+
+```ngql
+UPDATE EDGE <src_vid> -> <dest_vid> [@rank] OF <edge_type> SET <update_columns> [WHEN <condition>] [YIELD <columns>]
+```
+
+Use `UPDATE EDGE` to update properties on an edge. The `UPDATE EDGE` statement only updates one edge type of an edge at a time.
```

Comment:
The `UPDATE EDGE` statement only updates one edge type of an edge at a time.
=>
An `UPDATE EDGE` statement can only update the value of one property on an edge.

Reply:
Only one edge type can be updated at a time with the `UPDATE EDGE` statement? 

I think we should emphasize the "only one edge type" in this sentence, so the inactive voice is OK here.

Reply:
The `UPDATE EDGE` statement only updates one edge at a time.
I prefer this

Reply:
> The `UPDATE EDGE` statement only updates one edge at a time.
> I prefer this

one edge type, not one edge. And, in your version, I think "updates only one edge" is clearer. "only" should be next to the word that it modifies.